csrc/device_lower/pass/index.cpp

```diff
@@ -650,7 +650,7 @@ void IndexLowering::handleSerialGridReduction(
   }
   auto work_buffer_domain = IrBuilder::create<TensorDomain>(work_buffer_root);
   auto work_buffer_tv = IrBuilder::create<TensorView>(
-      work_buffer_domain, out_tv->dtype(), MemoryType::Global);
+      work_buffer_domain, DataType::Half, MemoryType::Global);
```

Comment:
Placeholder. This will be removed once we update the interface for requesting serial grid reduction to also specify the precision.